e2e-tests/cypress/integration/common/cookies.js

```diff
@@ -1,10 +1,31 @@
 import { Given, When, Then } from 'cypress-cucumber-preprocessor/steps';
-import {
-  expectCookieIsNotDefined,
-  expectCookiePolicy,
-  expectExpressSessionCookieIsDefined,
-} from '../cookies/cookies';
 import { DEFAULT_COOKIE_POLICY } from '../../../../packages/forms-web-app/src/lib/cookies';
+import { expectCookieIsNotDefined, expectCookiePolicy, expectExpressSessionCookieIsDefined } from '../cookies/cookies';
+
+When('the user navigates through the service', () => {
+  cy.goToLandingPage();
+  cy.goToTaskListPage();
+});
+
+When('the user accepts not necessary cookies', () => {
+  // To do
+});
```

Comment:
button click: `data-cy = cookie-banner-accept-analytics-cookies`

this should then display the accepted banner:

`data-cy = cookie-banner-accepted`